Source/CBLDatabase+Insertion.m

```diff
@@ -228,8 +228,8 @@ - (CBLStatus) forceInsert: (CBL_Revision*)inRev
         NSString* prevRevID = history.count >= 2 ? history[1] : nil;
         if (![self processAttachmentsForRevision: updatedRev
                                        prevRevID: prevRevID
-                                          status: &status])
-        return status;
+                                          status: &status] && CBLStatusIsError(status))
```

Comment:
Why is this necessary? -processAttachmentsForRevision shouldn't return NO unless there's an error. (If it does, we should fix that instead of testing for it here.)